django/http/response.py

```diff
@@ -240,7 +240,7 @@ def make_bytes(self, value):
         return force_bytes(value, self.charset)
 
     # These methods partially implement the file-like object interface.
-    # See http://docs.python.org/lib/bltin-file-objects.html
+    # See https://docs.python.org/2/library/stdtypes.html#file-objects
```

Comment:
the link for Python 3 might be https://docs.python.org/3/library/io.html#io.IOBase